Formula/pspg.rb

```diff
@@ -0,0 +1,27 @@
+class Pspg < Formula
+  desc "Unix pager optimized for psql"
+  homepage "https://github.com/okbob/pspg"
+  url "https://github.com/okbob/pspg/archive/0.5.tar.gz"
+  sha256 "754d1e380d072517e9bc2c3c38785e2f19a9f927f061de9a646fd1094baa204e"
+  head "https://github.com/okbob/pspg.git"
+
+  depends_on "ncurses"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def caveats; <<~EOS
+    Add the following line to your psql profile (e.g. ~/.psqlrc)
+         \\setenv PAGER pspg
```

Comment:
Any chance this can be a 2 space indent or is the 4 space indent required in `~/.psqlrc`?

Reply:
It's not required at all. I think I copy-pasted this structure from `asdf`'s formula.

Reply:
Alright, unless there's a reason for it let's stick to a 2 space indent here. Mixing & matching indentation where not strictly required is kinda gross 🙈.

Reply:
Done. (Though in fairness this indentation is for display purposes and not code. Homebrew might do well do establish a style for this sort of command output; right now there's vast inconsistency within the codebase (I checked `postgresql` for another example and it's a mess)).